npc/airports/airships.txt

```diff
@@ -288,44 +296,44 @@ airplane,250,58,2	script	Airship Staff#airplane	1_F_02,{
 		mes("Welcome\r"
 			"to the Airship.");
 		mes("How may I help you?");
-		next;
+		next();
 		select("Do you have a passenger named Thierry?");
 		mes("[Airship Staff]");
 		mes("I am sorry, but I do not think that we have a passenger by that name.");
-		close;
+		close();
 	}
 	// Hugel quest end
 	mes("[Airship Staff]");
 	mes("Welcome\r"
 		"to the Airship.");
 	mes("How may I help you?");
-	next;
+	next();
 	switch (select("Using the Airship", "Captain's Cabin", "Facilities", "Cancel")) {
 	case 1:
 		mes("[Airship Staff]");
 		mes("When you see a broadcast\r"
 			"announcing that we have\r"
-			"arrived at your destination,\r"
+			"arrived at your destination, \r"
```

Comment:
Extra space inserted (I guess this is the fault of the script / find&replace function you used to add spaces between function arguments - you might want to review it so that it doesn't add them inside strings)